### PR TITLE
Right Caret Added in Start Scheduler Button

### PR DESCRIPTION
**Pull request by 0xSaksham**
this PR tries to implement #319 

I still need to make the scheduler run in normal mode, and work flawlessly with `npm start` and `npm run dev:start`

### CODE REVIEW

**Comment by 0xSaksham** [comment]
**Please don't merge this branch right now**

**Comment by theborakompanioni** [comment]
> **Please don't merge this branch right now**

Hey @0xSaksham!

PR has been converted to draft and reviewers removed! Please add them anytime you think the changes are ready for a review. :raised_hands:

**Comment by 0xSaksham** [comment]
image

**Comment by 0xSaksham** [comment]
You have to put 3 different addresses to make the scheduler start.

**Comment by 0xSaksham** [comment]
This is Ready to merge!

File: src/components/Jam.tsx [hunk 1]
```diff
@@ -283,10 +284,23 @@ export default function Jam({ wallet }: JamProps) {
     setAlert(undefined)
     setIsWaitingSchedulerStart(true)
 
+    //    const destinations = addressValueKeys(addressCount).map((key) => values[key])
```
Comment (theborakompanioni) [inline src/components/Jam.tsx:287]:
```suggestion
```

File: src/components/Jam.tsx [hunk 2]
```diff
@@ -283,10 +284,23 @@ export default function Jam({ wallet }: JamProps) {
...
+        minmakercount: 1,
+        makercountrange: [1, 0],
+        mixdepthcount: addressCount,
+        mintxcount: 1,
+        txcountparams: [1, 0],
+        timelambda: 0.025, // 0.025 minutes := 1.5 seconds
+        stage1_timelambda_increase: 1.0,
+        liquiditywait: 13,
+        waittime: 0.0,
+      },
```
Comment (theborakompanioni) [inline src/components/Jam.tsx:303]:
```suggestion
```

File: src/components/Jam.tsx [hunk 3]
```diff
@@ -339,9 +353,12 @@ export default function Jam({ wallet }: JamProps) {
 
   return (
     <>
+      {' '}
+      {
+        // MARK: Scheduler Page
+      }
```
Comment (theborakompanioni) [inline src/components/Jam.tsx:359]:
```suggestion
```

**Comment by 0xSaksham** [comment]
Completed the Changes

**Review by theborakompanioni — APPROVED** [review]

**Pull request merged**